### PR TITLE
Db schema

### PR DESCRIPTION
**Pull request by TheRealBKJ**
## Description
This PR merges the Database schema for articles being put in the database into the AWS comprehend branch - which fetches the news data and then parses the titles through AWS comprehend to be evaluated for sentiment.

https://docs.google.com/document/d/1f1F0ui0pLvHa_9mdearynn-g3xRrlREDbUM2jluzVuc/edit?tab=t.0
https://docs.google.com/document/d/1caxzRLiKQk0_MtFrnqomYi177tcNThhMq8kmBgMRzAQ/edit?usp=sharing

Working towards users can be notified of important news and be shown this on website

## Test Plan

Just showing that was able to add and filter out articles with benchmark score of 0 using test data and filter out by ratio of negative or positive. Only articles with benchmakr > 0 and pos to neg ratio of 3 or 1/3 above
Screenshot 2025-07-11 at 2 51 30 PM

Screenshot 2025-07-11 at 2 49 58 PM

### CODE REVIEW

File: news-cron-server/prisma/schema.prisma [hunk 1]
```diff
@@ -13,3 +13,24 @@ datasource db {
...
+  title     String
+  keywords  String // for parsing for nodemailer to find which symbol its for
+  image_url String
+
+  // SentimentScore breakdown
+  sentiment     String
+  mixedScore    Float
+  negativeScore Float
+  neutralScore  Float
+  positiveScore Float
```
Comment (Ojasvijhamb) [inline news-cron-server/prisma/schema.prisma:29]: Nit: A good practice is to add a comment about where this data is collected from, and what it will be used for!

File: news-cron-server/prisma/schema.prisma [hunk 2]
```diff
@@ -13,3 +13,24 @@ datasource db {
...
+
+  sent        Boolean
```
Comment (Ojasvijhamb) [inline news-cron-server/prisma/schema.prisma:31]: Nit: change to is_sent, or better yet: is_processed.

File: news-cron-server/prisma/schema.prisma [hunk 3]
```diff
@@ -13,3 +13,24 @@ datasource db {
...
+  createdAt   DateTime @default(now())
+  publishedOn DateTime @map("PUBLISHED_ON") // convert date frmo api in to JS date
+
+  @@unique([guid])
```
Comment (Ojasvijhamb) [inline news-cron-server/prisma/schema.prisma:35]: Add a unique index instead of a constraint. This would be more performant when you search an article by GUID (for a potential future use-case where users want to see the sentiment score of a particular article for example).

**Review by Ojasvijhamb — COMMENTED** [review]
Good work here!
